create_deposition/README.md

```diff
@@ -0,0 +1,42 @@
+# Action for Creating Zenodo-Depostion
+This action creates a deposition with the metadata provided in `metadata.json` file.
+
+## Usage
+To create a deposition,first add the metadata to `metadata.json` file in the root directory. 
```

Comment:
The file can be in any path, users specify it via the `ZENODO_METADATA` variable